index.js

```diff
@@ -77,6 +74,24 @@ class ClientServiceBase {
     return 'rpc.status-bin';
   }
 
+  /**
+   * get max age
+   * INT_MAX is equivalent to java's Integer.MAX_VALUE
+   * @return {number}
+   */
+  static get maxAge() {
+    const INT_MAX = 2147483647;
```

Comment:
It's up to reviewers, but 2147483647 still seems to be something magic.
E.g., ``const INT32_MAX = 0x7fffffff`` might make sense more.